pkg/worker/worker_test.go

```diff
@@ -38,6 +38,11 @@ func (suite *WorkerSuite) stubFetchUserInfo(ctx context.Context, username string
 func (suite *WorkerSuite) SetupTest() {
 	err := suite.testConfigs.Store.TruncateAllTablesDANGEROUS(suite.testConfigs.Logger)
 	assert.NoError(suite.T(), err)
+
+	//GET USER ACCOUNT EACH TIME!
+	princ := getTestPrincipal(suite.testConfigs.Store, suite.testConfigs.UserInfo.EuaUserID)
+	suite.testConfigs.Principal = princ
```

Comment:
Should we remove the code in `GetDefaultTestConfigs` that populates this object since we're doing it here before each test? I could go either way, since it doesn't seem like it's harmful to keep (other than maybe a bit misleading)

Reply:
I think you are right, I was a little on the fence about it too, as it doesn't actively hurt anything, but I agree it's misleading. I'll refactor this

Reply:
Thanks Clay, I implemented this. I could see reverting back, even though it is a bit redundant, just because GetDefaults is supposed to be everything you need to run the test. It also revealed a few instances where the method was being called in the test instead of referencing the configuration object.

Reply:
Yeah - if we want to keep it split, we could pull the `principal` object and put it right on the `suite`, that way each test could reference `suite.testPrincipal` without going through the `testConfigs` object (and therefore not needing to create it in `GetDefaultTestConfigs`, but I think that might be a bit much for now